tensorflow_datasets/scripts/update_checksums.py

```diff
@@ -0,0 +1,108 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Script to update all the checksums.tsv files with tabs instead of spaces"""
+
+import os
+from absl import app
+from typing import List
+
+import tensorflow.compat.v2 as tf
+
+_DATASET_TYPES = list(['audio',
+                        'image',
+                        'image_claffication',
+                        'onject_detection',
+                        'question_answering',
+                        'structured',
+                        'summarization',
+                        'text',
+                        'translate',
+                        'video'])
+
+_ROOT_DIR = os.path.normpath(os.path.dirname(os.path.dirname(__file__)))
+
+_TXT_CHECKSUM_DIR = os.path.join(_ROOT_DIR, 'url_checksums')
```

Comment:
Rather than hardcoding path here. Is it not possible to reuse the constant defined in `checksums.py` ?

Reply:
@Conchylicultor The `_ROOT_DIR` used in `checksums.py` uses the following code
`_ROOT_DIR = os.path.normpath(os.path.join(os.path.dirname(__file__), '../..'))`
and thus, gets the absolute path upto the `../tensorflow_datasets` directory as the file is located in `../tensorflow_datasets/core/download`
But the `update_checksums` script is located in the `../tensorflow_datasets/scripts` and thus, has one level less in its path.
Therefore, I don't think we can use the same code for both of them unless we put the `update_checksums` script inside the `core` directory or the `scripts/cleanup` directory to make the path levels equal